index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
```

Comment:
The document seems to be written in English language so you should include ```lang="en" ```. This show helps render your file properly